packages/js/product-editor/src/components/attribute-control/new-attribute-modal.tsx

```diff
@@ -254,7 +256,11 @@ export const NewAttributeModal: React.FC< NewAttributeModalProps > = ( {
 											getProductAttributeObject(
 												value
 											) as EnhancedProductAttribute;
-										if ( termsAutoSelection === 'all' ) {
+										if (
```

Comment:
By default, the number of [items we get now is 10](https://woocommerce.github.io/woocommerce-rest-api-docs/#list-all-attribute-terms) (that's the `per_page` default value). So it seems like we won't see more than 10 terms here, because we are not sending the `per_page` prop [here](https://github.com/woocommerce/woocommerce/blob/trunk/packages/js/product-editor/src/components/attribute-control/new-attribute-modal.tsx#L245-L251) and [here](https://github.com/woocommerce/woocommerce/blob/trunk/packages/js/product-editor/src/components/attribute-term-input-field/attribute-term-input-field.tsx#L86-L89).